### PR TITLE
Fixed events not loading users on Admin edit page

### LINKED ISSUE #183: Editing an Event removes associated speakers and companies

### PR DESCRIPTION
**Pull request by Naapperas**
Closes #183

### CODE REVIEW

File: app/Http/Controllers/CRUDController.php [hunk 1]
```diff
@@ -68,6 +69,16 @@ protected function with(): array
...
 
+    /**
+     * The associated relations to load when loading the model.
+     *
+     * @return array<mixed, mixed>
+     */
+    protected function load(): array
+    {
+        return [];
+    }
```
Comment (ttoino) [inline app/Http/Controllers/CRUDController.php]: What do you think about making this a property, since it doesn't take any arguments?

**Review by ttoino — DISMISSED** [review]

File: app/Http/Controllers/CRUDController.php (hunk 1 above)
Comment (Naapperas) [reply app/Http/Controllers/CRUDController.php]: I was following the same principle used on `with`, I can change it but I believe they should follow the same "pattern". Thoughts?

**Comment by Naapperas** [comment]
Also forgot to mention, but as a bonus I moved the relation loading from the EventDay model to the Program page which greatly reduced the number of models loaded when visiting the Home page.

File: app/Http/Controllers/CRUDController.php (hunk 1 above)
Comment (ttoino) [reply app/Http/Controllers/CRUDController.php]: The `with` function needs dynamic data, while this one doesn't, but I get what you mean, I'll leave it for you to decide.

Comment (Naapperas) [reply app/Http/Controllers/CRUDController.php]: Ok, that makes sense. It's not that big of a deal.

**Review by ttoino — APPROVED** [review]

**Review by ttoino — APPROVED** [review]

**Pull request merged**